### PR TITLE
Emit `_fltused` on `uefi` targets as a short-term workaround

### PR DESCRIPTION
**Pull request by oli-obk**
fixes https://github.com/rust-lang/rust/issues/62785

### CODE REVIEW

File: ci/docker/x86_64-unknown-uefi/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:18.04
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    gcc libc6-dev ca-certificates
```
Comment (alexcrichton) [inline ci/docker/x86_64-unknown-uefi/Dockerfile]: Is this intended to be hooked up to CI? (it doesn't happen automatically, extra configuration would be needed)

Comment (josephlr) [reply ci/docker/x86_64-unknown-uefi/Dockerfile]: @oli-obk can we do CI configs in a follow up PR, so that this fix can get merged?

**Comment by pnkfelix** [comment]
This is blocking rust-lang/rust#62785

@alexcrichton can you respond to @josephlr 's question regarding handling of CI?

**Comment by alexcrichton** [comment]
I've removed the CI changes for now.

**Comment by alexcrichton** [comment]
I've published 0.1.20 with this change to get pulled into the compiler

**Pull request merged**